### ISSUE TITLE
[Mac] Database Navigator Filter Field Clear (X) Button Does Not Work

### ISSUE THREAD

**Issue by RobertHana1** [dbeaver/dbeaver#21630]
### Description

Database Navigator Filter Field Clear (X) Button Does Not Work

### DBeaver Version

CE 23.2.2.2023*

### Operating System

Mac OS X Ventura 13.5.1

**Comment by MashaKorax** [comment]
Thanks for your bug report. I reproduced it. It works only after several clicks (5-7)

**Comment by LonwoLonwo** [comment]
Duplicate of https://github.com/dbeaver/dbeaver/issues/10590
